prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -2199,6 +2199,36 @@ void rxr_pkt_handle_rtr_recv(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry)
 	rxr_pkt_entry_release_rx(ep, pkt_entry);
 }
 
+static int efa_copy_from_hmem(char *data, struct efa_mr **buf_mr, int max_copy_size,
```

Comment:
can you move it to `efa_hmem.h`? 

`struct efa_mr **buf_mr` should be `desc`.

`data` should be `buff`, `max_copy_size`should  be `buff_size`.

Reply:
function name should be `efa_copy_from_hmem_iov`.

Reply:
Also add a doxygen function header.